kafka/event.go

```diff
@@ -145,6 +145,33 @@ func (o OAuthBearerTokenRefresh) String() string {
 	return "OAuthBearerTokenRefresh"
 }
 
+// ErrRebalance reports an application-level error occuring during a RebalanceCb call.
+func ErrRebalance(wrapped error) *ApplicationError {
```

Comment:
The naming of this method makes it look like a kafka.Error code, which is confusing.
Can't we just call it ApplicationRebalanceError?

Reply:
Done: I actually called it `NewApplicationRebalanceError` to be have a more idiomatic naming, since it returns a pointer to `ApplicationError`.